README.md

```diff
@@ -48,6 +48,8 @@ Run the following command:
 $ npm run test-screener
 ```
 
+After this command, `sauce-connect-launcher` version 4.5.4 will be installed.
```

Comment:
Let's remove this to simplify the docs for end users, as for screener users they won't know what this is

Reply:
Removed